src/pages/sign-up-index-page/student-form.js

```diff
@@ -0,0 +1,51 @@
+const formRef = document.querySelector('form');
+const getInputRef = (name) => formRef.querySelector(`[name="${name}"]`);
+
+export const StudentForm = {
+  get formData() {
+    return new FormData(formRef);
+  },
+
+  get data() {
+    return {
+      name: this.formData.get('name'),
+      surname: this.formData.get('surname'),
+      birthday: this.formData.get('birthday'),
+    };
+  },
+
+  set data({ name, surname, birthday }) {
+    getInputRef('name').value = name;
+    getInputRef('surname').value = surname;
+    getInputRef('birthday').value = birthday;
```

Comment:
This line overrides the default value from input provided in HTML, can you please check here for the birthday variable not to have empty value?